internal/service/loadbalancer/backend.go

```diff
@@ -4,366 +4,506 @@ import (
+							Default:             stringdefault.StaticString(""),
+							Validators: []validator.String{
+								stringvalidator.OneOf(
+									"",
+									string(upcloud.LoadBalancerProxyProtocolVersion1),
+									string(upcloud.LoadBalancerProxyProtocolVersion2),
+								),
+							},
+						},
+						"sticky_session_cookie_name": schema.StringAttribute{
+							MarkdownDescription: "Sets sticky session cookie name. Empty string disables sticky session.",
+							Optional:            true,
+							Computed:            true,
+							Default:             stringdefault.StaticString(""),
+							Validators: []validator.String{
+								stringvalidator.LengthBetween(0, 64),
+							},
+						},
+						"timeout_server": schema.Int64Attribute{
+							MarkdownDescription: "Backend server timeout in seconds.",
+							Optional:            true,
+							Computed:            true,
+							Default:             int64default.StaticInt64(10),
+							Validators: []validator.Int64{
+								int64validator.Between(1, 86400),
+							},
+						},
+						"timeout_tunnel": schema.Int64Attribute{
+							MarkdownDescription: "Maximum inactivity time on the client and server side for tunnels in seconds.",
+							Optional:            true,
+							Computed:            true,
+							Default:             int64default.StaticInt64(3600),
+							Validators: []validator.Int64{
+								int64validator.Between(1, 3024000),
+							},
+						},
+						"tls_enabled": schema.BoolAttribute{
+							MarkdownDescription: "Enables TLS connection from the load balancer to backend servers.",
+							Optional:            true,
+							Computed:            true,
+							Default:             booldefault.StaticBool(false),
+						},
+						"tls_use_system_ca": schema.BoolAttribute{
+							MarkdownDescription: "If enabled, then the system CA certificate bundle will be used for the certificate verification.",
+							Optional:            true,
+							Computed:            true,
+							Default:             booldefault.StaticBool(false),
+						},
+						"tls_verify": schema.BoolAttribute{
+							MarkdownDescription: "Enables backend servers certificate verification. Please make sure that TLS config with the certificate bundle of type authority attached to the backend or `tls_use_system_ca` enabled. Note: `tls_verify` has preference over `health_check_tls_verify` when `tls_enabled` in true.",
+							Optional:            true,
+							Computed:            true,
+							Default:             booldefault.StaticBool(false),
+						},
+					},
+				},
+				Validators: []validator.List{
+					listvalidator.SizeBetween(0, 1),
 				},
 			},
 		},
+		Version: 1,
 	}
 }
 
-func resourceBackendCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) (diags diag.Diagnostics) {
-	svc := meta.(*service.Service)
-	serviceID := d.Get("loadbalancer").(string)
+func (r *backendResource) ModifyPlan(ctx context.Context, req resource.ModifyPlanRequest, resp *resource.ModifyPlanResponse) {
```

Comment:
Is this needed? Looks like `p` is not used after getting an attribute value to it.

Reply:
Removed as unnecessary.